src/unix/uilocale.cpp

```diff
@@ -21,6 +21,10 @@
 #if wxUSE_INTL
 
 #include <locale.h>
+#ifdef __APPLE__
+    #include <xlocale.h>
+#endif
+
```

Comment:
Do we still need this if this is not compiled under Mac any longer?

Reply:
Oops, yes, sorry.

Thinking more about this, I believe we ought to use

```cpp
#ifdef HAVE_XLOCALE_H
    #include <xlocale.h>
#endif
```

here for consistency and because this may be needed under other systems too. Let me change it like this...